package/python-hvac/python-hvac.mk

```diff
@@ -0,0 +1,12 @@
+################################################################################
+#
+# python-hvac
+#
+################################################################################
+
+PYTHON_HVAC_VERSION = 0.10.4
+PYTHON_HVAC_SOURCE = v$(PYTHON_HVAC_VERSION).tar.gz
+PYTHON_HVAC_SITE = https://github.com/hvac/hvac/archive
+PYTHON_HVAC_SETUP_TYPE = setuptools
+
```

Comment:
all good - just add in the LICENSE_FILE and LICENSE type for this to be complete-ish.